src/main/java/core/basesyntax/Info.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Info {
+    public void moreInfo();

```

Comment:
The 'public' modifier is redundant for interface methods. Interface methods are public by default, so you can remove the 'public' keyword.